packages/ui-core/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cnCreate from 'utils/cnCreate';
+import './Tile.less';
+
+export enum Theme {
+    LIGHT = 'light',
+    DARK = 'dark',
+}
+
+export enum Radius {
+    DEFAULT = 'default',
+    ROUNDED = 'rounded',
+}
+
+export enum Shadow {
+    ZERO = 'zero',
+    LOW = 'low',
+    HIGH = 'high',
+    HOVER = 'hover',
+}
+
+export interface ITileProps {
+    /** Link href */
+    href?: string;
+    /** Tile view according to the background color */
+    theme?: Theme;
+    /** Border radius */
+    radius?: Radius;
+    /** Shadow levels of tile contour */
+    shadowLevel?: Shadow;
+    /** Enabling shadow on hover */
+    isInteractive?: boolean;
+    /** Custom class */
+    className?: string;
+    /** Handler */
+    onClick?: (e: React.MouseEvent<HTMLDivElement>) => void;
+}
+
+const cn = cnCreate('mfui-tile');
```

Comment:
```suggestion
const cn = cnCreate('mfui-beta-tile');
```
Забыл про это упомянуть. Чтобы первая и вторая версия либы в проектах не конфликтовали из-за измененых стилей. А после релиза второй версии можно было во всем проекте заменить.